matrix/app_server/deploy_app.py

```diff
@@ -358,13 +359,28 @@ async def get_one_endpoint() -> str:
             host = random.choice(ips)
             return host
 
-    return asyncio.run(
-        query(
-            get_one_endpoint,
-            output_jsonl,
-            input_jsonls,
-            model=metadata["model_name"],
-            app_name=metadata["name"],
-            **kwargs,
+    app_type = metadata["app_type"]
+    if app_type in ["llm", "sglang_llm"]:
+        from matrix.client.query_llm import main as query_llm
+
+        return asyncio.run(
+            query_llm(
+                get_one_endpoint,
+                output_jsonl,
+                input_jsonls,
+                model=metadata["model_name"],
+                app_name=metadata["name"],
+                **kwargs,
+            )
+        )
+    else:
+        from matrix.client.execute_code import main as execute_code
```

Comment:
maybe good do:
```python
elif app_type in ["code", "hello"]:
    ...
else:
    raise ValueError(f"Unknown app_type {app_type}")
```

in case we have other app types in the future and forgot to change it here?